### DISCUSSION TITLE
"обнаружен вирус"

### DISCUSSION THREAD

**Discussion by fosterriko** [bol-van/zapret#846] [Q&A]
Странно: скачивал первый файл - всё было нормально, сейчас не скачивает ни одну версию ни из одного браузера. В двух говорит "прервано, обнаружен вирус" в третьем "не удалось". Пробовал понижать настройки безопасности - не работает. Никто не пишет о такой проблеме, где я оказался глупи?

**Comment by HentayHentay** [comment]
Скорее скачивал версию под MS ? 
Если да, то [сюда](https://github.com/bol-van/zapret/issues/718#issuecomment-2460457760)
И  при усилии, что отсюда брал инструмент

**Comment by Fenrir-0xFF** [comment]
> Странно: скачивал первый файл - всё было нормально, сейчас не скачивает ни одну версию ни из одного браузера. 

Судя по фразе "не скачивает ни одну версию" возможно и не под винду. 
Если у тебя яндекс браузер или антивирь касперский или прочее дерьмо которое лицензировано в рашке и которое навязывают провайдеры то не удивительно, поскольку РКН недавно начял бороться с zapret'ом 

Ктомуже бинакрики упакованы UPX'ом чтобы влазили на роутеры где мало памяти, раньше антивири чясто на упакованные исполнительные файлы агрились, но потом конкретно для UPX'а сделали исключение. 
Выруби антивирь, скачяй файлы и закинь на роутер, если под винду то добавь исключения в антивирь.
